src/types.db

```diff
@@ -148,6 +149,9 @@ node_octets             rx:DERIVE:0:U, tx:DERIVE:0:U
 node_rssi               value:GAUGE:0:255
 node_stat               value:DERIVE:0:U
 node_tx_rate            value:GAUGE:0:127
+nr_overcommit_hugepages value:DERIVE:0:U
+nr_hugepages            value:DERIVE:0:U
```

Comment:
Hi @octo are you asking to report the used pages alongside the total, or only the used pages? the plugin is reporting the free pages as well as the total already and the used value could be deduced. the use case we're looking it is leveraging this information as is for intelligent workload placement so it would be nice to know what's available on the system not only what is used... what do you think?